debugger/debugrt/host/host.c

```diff
@@ -77,6 +155,13 @@ OE_NO_OPTIMIZE_END
 
 #endif
 
+/**
+ * The version of the debugger contract supported by the runtime.
+ * For development purposes, this value can be overridden by setting
+ * the OE_DEBUGGER_CONTRACT_VERSION enviroment variable.
+ */
+uint32_t oe_debugger_contract_version = 1;
```

Comment:
This is different from the PR description, which says the version is 0 by default.

Reply:
If we have backward compatibility, is versioning still necessary?

Reply:
> This is different from the PR description, which says the version is 0 by default.

Fixed the outdated PR description. Good catch.

Reply:
> If we have backward compatibility, is versioning still necessary?

Yes. This will allow an older debugger to determine that it cannot completely handle an application that supports a newer version of the contract.

Say in contract v2 we support SGX 2 which allows creating TCS dynamically.
A debugger that understands only contract v1 can detect this and inform the user to use a newer version of the debugger. The older debugger may still be able to partially debug the application.